changelogs/fragments/2824-gitlab_project-project-under-user.yml

```
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - User projects are created using namespace ID, instead of user ID.
```

Please adjust the changelog fragment:
```suggestion
  - gitlab_project - user projects are created using namespace ID, instead of user ID (https://github.com/ansible-collections/community.general/pull/2824).
```
Same for the other line.

---

Should be fixed in the latest commit